.travis.yml

```diff
@@ -19,17 +20,27 @@ rvm:
 matrix:
   exclude:
   - rvm: 2.3
-    env: RAILS_VERSION=6.0 SORBET_VERSION=0.4.4386
+    env: RAILS_VERSION=6.0
   - rvm: 2.4
-    env: RAILS_VERSION=6.0 SORBET_VERSION=0.4.4386
+    env: RAILS_VERSION=6.0
+  # only need to have 1 test with specific sorbet version
   - rvm: 2.3
-    env: RAILS_VERSION=5.2 # SORBET_VERSION=0.4.4386
+    env: RAILS_VERSION=5.2 SORBET_VERSION=0.4.4386
   - rvm: 2.4
-    env: RAILS_VERSION=5.2 # SORBET_VERSION=0.4.4386
+    env: RAILS_VERSION=5.2 SORBET_VERSION=0.4.4386
   - rvm: 2.6
-    env: RAILS_VERSION=5.2 # SORBET_VERSION=0.4.4386
+    env: RAILS_VERSION=5.2 SORBET_VERSION=0.4.4386
   - rvm: ruby-head
-    env: RAILS_VERSION=5.2 # SORBET_VERSION=0.4.4386
+    env: RAILS_VERSION=5.2 SORBET_VERSION=0.4.4386
+  # only need to have 1 test with srb init
+  - rvm: 2.3
```

Comment:
Given the intention, would this be better as an `include:`, rather than adding to `matrix:` and excluding all but one?

Reply:
👍